playground/src/app/simple-call/simple-call.component.ts

```diff
@@ -56,6 +57,15 @@ export class SimpleCallComponent implements OnInit, OnDestroy {
       eval(wrapper);
     }
 
+    if (this.extras !== "{}" && this.extras !== "") {
+      const extrasJson = JSON.parse(this.extras);
+      const extraKeys = Object.keys(extrasJson);
+
+      for (const key of extraKeys) {
+        this.client?.extra.set(key, extrasJson[key]);
+      }
+    }
```

Comment:
primeira sugestão: `this.extras !== ""` por `this.extras` (`!!"" === false`)

segunda sugestão: ter um try/catch nessa condição pra deixar o erro bonito caso dê problema (talvez esse bloco todo possa ser movido pra dentro do try/catch lá de baixo?)